docs/release.md

```diff
@@ -51,11 +51,59 @@ _Note 1_: Syncing translations (first workflow) is an optional step which means
 
 ### Production
 
-For releasing production, you need to run `Production Release` workflow, it will pull the latest translation changes on `next` branch and checkout to `next` branch and pull the latest changes then it tries to merge the `next` into `main` by `--no-ff` strategy, To make sure that a new commit is made And previous commits that may have `[skips ci]` Do not prevent workflow from triggering.
+#### Overview
+
+Follow these steps for the release:
+
+1. Run the `Production Release` workflow manually.
+2. When workflow finished, make sure you've updated changelog and created a PR, as it explained below.
+3. Run the `Deploy` workflow if the publish was successful.
+4. Promote our clients (widget and playground) to production on Vercel (ask the team if you don't have access).
+5. Run `yarn post-release` to merge main into next to make sure they are in sync.
+
+**NOTE 1:**
+
+Ensure you send a highlight note on Telegram [like this](https://t.me/c/1797229876/15255/23609) at the end.
 
+**NOTE 2:**
+
+Ensure you update widget-examples using `yarn add @rango-dev/widget-embedded@latest`. Then open a new PR on the repo to ensure all examples are on the latest version.
+
+#### Details
+
+For releasing production, you need to run `Production Release` workflow, it will pull the latest translation changes on `next` branch and checkout to `next` branch and pull the latest changes then it tries to merge the `next` into `main` by `--no-ff` strategy, To make sure that a new commit is made And previous commits that may have `[skips ci]` Do not prevent workflow from triggering.
 
 _Note 1_: Make sure you are having permission for `push` on `main`.
```

Comment:
We do not want permission for push on `main`
We only want it for push on `next` after release.

Reply:
updated.